apigateway/helm/templates/kibana.yaml

```diff
@@ -113,11 +118,16 @@ spec:
                   name: {{ include "apigateway.kibanatruststorepassword" . }}
                   key: password
             {{- end }}
+          {{- if .Values.kibana.readinessProbe }}
+          readinessProbe:
+            {{- toYaml .Values.kibana.readinessProbe | nindent 12 }}
+          {{- else }}
           readinessProbe:
```

Comment:
readyness probe in values.yaml, all defaults in values.yaml, add optioal lifenessprobe

Reply:
new commit:
- moved default readinessProbe in values.yaml
- added optional livenessPobe